lighthouse/tests/validator_client.rs

```diff
@@ -499,3 +499,27 @@ fn latency_measurement_service() {
             assert!(!config.enable_latency_measurement_service);
         });
 }
+
+#[test]
+fn validator_registration_batch_size() {
+    CommandLineTest::new().run().with_config(|config| {
+        assert_eq!(config.validator_registration_batch_size, 500);
+    });
+    CommandLineTest::new()
+        .flag("validator-registration-batch-size", Some("100"))
+        .run()
+        .with_config(|config| {
+            assert_eq!(config.validator_registration_batch_size, 100);
+        });
+}
+
+#[test]
+#[should_panic]
+fn validator_registration_batch_size_zero_value() {
+    CommandLineTest::new()
+        .flag("latency-measurement-service", Some("0"))
+        .run()
+        .with_config(|config| {
+            assert!(!config.enable_latency_measurement_service);
```

Comment:
I guess you're meant to test the `validator-registration-batch-size` flag?

Reply:
Ooh, good catch! Thanks 🙏